provision/testdata/list_flavors.stdout.txt

```diff
@@ -1,5 +1,7 @@
 INFO: Available configuration flavors:
 INFO: openshift-4.8-baremetal:	Red Hat OpenShift Container Platform 4.8 on Baremetal
+INFO: openshift-4.9-openstack:	Red Hat OpenShift Container Platform 4.9 on OpenStack
+INFO: openshift-4.8-openstack:	Red Hat OpenShift Container Platform 4.8 on OpenStack
```

Comment:
The ordering is broken here. It seems that we had earlier followed the convention of grouping all the releases of a given integration together (meaning all BMs together, all OpenStacks together, etc.). I believe this was not followed correctly during the BM flavor change, so lets fix it now.

Reply:
sure sumit